app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
```

Comment:
- [ ] The post model should have a method that returns the 5 recent comments per post, kindly consider implementing this so you can return limited posts in your app

Reply:
Updated